src/main/kotlin/com/exactpro/th2/conn/amqp/client/AmqpClient.kt

```diff
@@ -64,23 +64,19 @@ class AmqpClient(config: Config, val errorReporter: (Throwable) -> Unit) : IClie
         LOGGER.info("Queue consumer created to read data form the Queue:  {}", receiveDestination)
 
         producer = jmsContext.createProducer() as JmsProducer
-
-        producer.setProperty(sessionAliasPropertyName, sessionAlias);
     }
 
     override fun setMessageListener(callback: (ByteArray) -> Unit) {
         LOGGER.debug("Set an asynchronous queue listener")
         consumer.messageListener = MessageListener { message: Message ->
-            if (sessionAlias == message.getStringProperty("sessionAlias")) {
-                LOGGER.debug("Message received from the Queue:  {}", receiveDestination.toString())
-                message.runCatching(Companion::toBytes).onSuccess(callback).onFailure {
-                    LOGGER.error(it) {"Error while processing incoming message"}
-                    errorReporter(it)
-                }
-                message.runCatching(Message::acknowledge).onFailure {
-                    LOGGER.error(it) {"Error while acknowledging received message"}
-                    errorReporter(it)
-                }
+            LOGGER.debug("Message received from the Queue:  {}", receiveDestination.toString())
```

Comment:
```suggestion
            LOGGER.debug {
                "Message received from the Queue: $receiveDestination")
            }
```
LOGGER is Kotlin logger defined via `KotlinLogging.logger { AmqpClient::class.simpleName }` and it doesn't support the `{}` formatting unlike slf4j